packages/web/src/components/gcds-pagination/test/gcds-pagination.e2e.ts

```diff
@@ -5,9 +5,7 @@ describe('gcds-pagination', () => {
   it('renders', async () => {
     const page = await newE2EPage();
     await page.setContent(`
-      <gcds-pagination
-       display="simple" 
-      ></gcds-pagination>
+      <gcds-pagination display="simple"></gcds-pagination>
     `);
 
```

Comment:
I can't highlight the exact line but since this component is now using shadow-dom the 'renders url' test will need to be updated.

The only change needed is [this line](https://github.com/cds-snc/gcds-components/blob/4c49718b8ed0d10b96708b6ad637ad827e302452/packages/web/src/components/gcds-pagination/test/gcds-pagination.e2e.ts#L42), it will need to be changed to `const element = await page.find('gcds-pagination >>> [aria-current*=page]');`